site/src/request_handlers/self_profile.rs

```diff
@@ -526,7 +526,7 @@ pub async fn handle_self_profile(
         .benchmark(selector::Selector::One(bench_name.to_string()))
         .profile(selector::Selector::One(profile.parse().unwrap()))
         .scenario(selector::Selector::One(scenario))
-        .metric(selector::Selector::One(Metric::CpuClock));
+        .metric(selector::Selector::One(Metric::InstructionsUser));
```

Comment:
This will break all historical queries for self-profile data, right? Do we want to perhaps query for both or some other mechanism to not do that?

Reply:
An alternative could be to first refactor self profile management (https://github.com/rust-lang/rustc-perf/issues/1896), so that we can e.g. version it, and then have some logic to detect if the archive has the old (walltime) or new (icount) version, and perform the query base on that.

But I'm not sure if people really look at old self-profile results at all, usually we're only interested in them for a brief moment after/before a PR is merged.